ql/ql/test/callgraph/Foo.qll

```diff
@@ -10,9 +10,7 @@ class Foo extends AstNode {
   predicate baz() { bar() }
 }
 
-class Sub extends Foo {
```

Comment:
You should revert the change to this file.  
It's a QL-for-QL test file. 

(Or you could make that override non-redundant, and update the expected test output).